### PR TITLE
Complete PS Update to UriBeacon

### PR DESCRIPTION
**Pull request by roywant**
Addressing remaining review issues

### CODE REVIEW

**Comment by roywant** [comment]
@schilit 
Experimented with the Windows version of GitHub and tried out various features. The outstanding merge contains improvements based on the BlueGiga review

File: beacons/BLED112/cdc.xml [hunk 1]
```diff
@@ -1,20 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-
```
Comment (schilit) [inline beacons/BLED112/cdc.xml]: copyright notice?

File: beacons/BLED112/config.xml [hunk 2]
```diff
@@ -1,19 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
```
Comment (schilit) [inline beacons/BLED112/config.xml]: copyright notice here and elsewhere?

**Comment by roywant** [comment]
@schilit Added the copyright notices

**Pull request merged**